manifests/overlays/prod/configs/argo_rbac_cm/policy.csv

```diff
@@ -61,4 +61,12 @@ p, role:aiops-admin, applications, sync, aiops/*, allow
 p, role:aiops-admin, applications, override, aiops/*, allow
 p, role:aiops-admin, applications, action/*, aiops/*, allow
 g, aicoe-aiops-devops, role:argocd-admin
-g, aicoe-aiops-devops, role:aiops-admin
\ No newline at end of file
+g, aicoe-aiops-devops, role:aiops-admin
+
+p, role:ccx-admin, applications, get, ccx/*, allow
+p, role:ccx-admin, applications, update, ccx/*, allow
+p, role:ccx-admin, applications, delete, ccx/*, allow
+p, role:ccx-admin, applications, sync, ccx/*, allow
+p, role:ccx-admin, applications, override, ccx/*, allow
+p, role:ccx-admin, applications, action/*, ccx/*, allow
+g, ccx-dev, role:ccx-admin
```

Comment:
I think argocd admins should have "get" access to this project as well.

Reply:
Good call. I think aiops should have access to it as well. :slightly_smiling_face:

Reply:
Actually, argocd admins shoul have access to all applications, no matter what project. We should add a generic rule for that.

Reply:
Additionally argocd-admins already have access to all projects:

https://github.com/AICoE/aicoe-cd/blob/704dc0366b63ac873ff9303ad47d1f0ca7e2654a/manifests/overlays/prod/configs/argo_rbac_cm/policy.csv#L10-L12